database/seeders/0003.admin.js

```diff
@@ -0,0 +1,44 @@
+const { v4: uuid } = require('uuid');
```

Comment:
we can remove that seeder, because I have an unpublished branch with the new vocascan cli, where it is possible to create users with admin rights from the cli.